app/controllers/items_controller.rb

```diff
@@ -1,6 +1,8 @@
 # Provides full CRUD to Items. Every item is rooted in a BaseItem, but Diaperbanks have full control to do whatever
 # they like with their own Items.
 class ItemsController < ApplicationController
+  before_action :check_for_existing_item_name, only: [:create], if: :item_params_valid?
```

Comment:
I feel like really what we want to do here is have a validation on uniqueness, no? You can only have one item per organization with the same name.

Reply:
Ahhhh that makes total sense. There is already a validation on uniqueness, so I will go off that. I initially went that route but ran into some issues. I'm going to try again and I'll be sure to reach out if I have issues!